.github/workflows/add-to-ao-project.yml

```diff
@@ -0,0 +1,21 @@
+name: Add issues to Actionable Observability project
+on:
+  issues:
+    types: [labeled]
+jobs:
+  sync_issues_with_table:
```

Comment:
Mind adding a condition here so it only runs when the label matches the team. [Here](https://github.com/elastic/kibana/blob/main/.github/workflows/add-to-apm-project.yml#L10) is an example.

Reply:
Added. Can you have a look and approve if everything's ok? Thanks